TYPO3.Flow/Classes/TYPO3/Flow/Security/Account.php

```diff
@@ -347,6 +360,51 @@ public function setExpirationDate(\DateTime $expirationDate = null)
     }
 
     /**
+     * @return integer
+     * @api
+     */
+    public function getFailedAuthenticationCount()
+    {
+        return $this->failedAuthenticationCount;
+    }
+
+    /**
+     * @param integer $failedAuthenticationCount
+     * @return void
+     */
+    public function setFailedAuthenticationCount($failedAuthenticationCount)
+    {
+        $this->failedAuthenticationCount = $failedAuthenticationCount;
+    }
```

Comment:
Hmm... good question, you are right... I would probably do it inside the "setAuthenticationStatus" method, as introducing the other methods would change a public API.

All tbe best,
Sebastian